biogrid/libs/biogrid-simulator/src/lib/config/efficiency.ts

```diff
@@ -53,6 +54,32 @@ export function calculatePower(voltage: number, resistance: number) {
   return (Math.pow(voltage, 2) / resistance);
 }
 
+/**
+ * This function calculates the power in the circuit for usage in the efficiency
+ * It calculates this based on the two physics equations
+ *    1. P = V * I
+ *    2. V = I * R
+ * where P is power, V is voltage, I is current and R is resistance
+ * Since current is constant in circuits
+ * Th final equation used is P = I^2 * R
+ * @param current is the current in the circuit at that particular time
+ * @param resistance is the resistance of the wires
+ * @return the power transport over the given @param resistance
+ */
+export function calculatePowerWithCurrent(current: number, resistance: number) {
+  return (Math.pow(current, 2) * resistance);
+}
+
+/**
+ * This function is used to calculate the current in the circuit using the given @param power
```

Comment:
nit: the same way you included the equation in the comment for calculatePowerWithCurrent, can you include it in this function comment as well?

Reply:
definitely, let me add that